config/longrun_configs/amip_edonly_1M.yml

```diff
@@ -1,12 +1,12 @@
 FLOAT_TYPE: "Float32"
 aerosol_radiation: true
 atmos_config_file: "config/longrun_configs/longrun_aquaplanet_allsky_1M.yml"
+checkpoint_dt: "31days"
```

Comment:
This run fails pretty quickly, so I'd like to have earlier checkpoints available